# Data description
Wheat (Triticum aestivum) is grown on over five million acres in the Pacific Northwest (PNW) as it is one of the only economically viable crops in areas of low or intermediate rainfall. Herbicide resistant varieties are important in growing wheat in the PNW due to the highly competitive weeds such as jointed goatgrass (Aegilops cylindrica) and cheatgrass (Bromus tectorum) which are the leading to losses in quality and quantity. Colorado State University (CSU) developed a new source of herbicide resistance conferring resistance to Quizalofop P herbicides in wheat. Due to winter wheat’s longer life cycle compared to spring wheat, CSU’s winter herbicide resistant variety will be crossed with Oregon State University’s (OSU) spring wheat and then to OSU’s winter wheat in an attempt to speed up the breeding process. This breeding scheme will be compared to a conventional breeding scheme, where CSU’s resistant varieties are directly crossed to OSU’s winter wheat varieties until the desired resistant winter wheat that is adapted to Oregon’s climate and stressors is developed. The goal is to study whether wheat breeding schemes can be sped up by using spring wheat to decrease the amount of time needed to create a new winter wheat variety.
	To compare the speed of the two breeding schemes, data will be recorded for the dates of planting for each generation, anthesis, and harvest. This data will be studied through observation and recorded in a table with date on the x-axis and the variety on the y-axis. This dataset will be about 1 MB. In addition, genotyping the end results for various genes (about 30) such as disease resistance will be used to compare the final products of the two schemes to see if anything was gained or lost using the speed breeding method. This information will be collected through phenol-chloroform DNA extraction and KASP-PCR for the various genes and then recorded in a table stating whether the gene was present or not. This dataset will be around 2 or 3 MB, potentially more if the exact PCR results are included. This experiment will occur over the course of about two years. The study started November 2017 and has a goal end date of November 2019. 

# Roles and responsibilities
1.I am in charge of keeping track of the dates of planting, harvest and anthesis. The two lab technicians I work closely with (Lab techs 1 and 2) and I are in charge of the genotyping data. Lab techs 1 and 2 and I are all in charge of the molecular marker data and primers to identify the traits we are selecting for. Overall I do most of the data collection since it is my thesis but Lab techs 1 and 2 also help when needed and we work together for data organization. Lab tech 1 primarily does instrumentation maintenance, software creation and maintenance if needed, and access control. Lab tech 2 helps with quality control (making sure the plants stay a live and healthy with fertilizer, herbicides etc.) in order for the data collection to go smoothly. These are the main rolls while we are collecting data.

When it comes to writing and publication, my PI will step in and help with the access control of the publication as well as quality control (looking over my writing). I will be doing the metadata generation. Archiving and preservation of data will go through my PI as well as through another one of his faculty assistants. 

2.Sensitive data is data that contains private data about human subjects that is not as restrictive as confidential data, however still is private information such as employee applications.  Normally this data is at a no or minimal risk level but should still be protected.

My data is unrestricted data and can be used for general use and easily found through databases or on the internet without posing a risk to anyone since my subjects are wheat plants.

3. I am on a graduate assistantship with my PI so my lab/OSU is funding my research. My PI does not have any data management or data sharing requirements. I simply work for my advisor year-round while performing my research, therefore no contracts or disclosure agreements other than my assistant ship contract. 

# Data standards and metadata
Our lab does not have a metadata standard. AgMES is provides access to bibliographic information on agricultural science and technology and was developed by the FAO. It has the goal of providing global public data in multiple languages through access to publications, journal articles, book chapters and more.

I have been using Zotero as a way to manage my data for reference articles. I can create files for different topics and chapters of my thesis. For the hard data, I will keep dates (planting, anthesis, harvest) on an EXCEL file that will include a section for notes or other metadata. These documents will be kept on my laptop as well as on the lab’s R drive where we share all of our information. There are no metadata requirements for my lab, when I reached out to faculty members they were a little confused on the topic. It is a general plant breeding program that does not perform a lot of research unless it is through the graduate students.

Zotero is a way to manage peer reviewed articles. Once downloaded, while browsing the wed or databases, when you come across a paper that you would like to save you click on the Zotero button on your internet browser. The button then sends the paper to the Zotero app on your computer. Where all of the papers of interest can be saved. It automatically downloads all of the citation information, which can also be edited if something is missing or incorrect etc. You can also set it up to Word of GooglDocs so you can quickly add references to your paper as you type instead of having to look 

# Storage and security
My data will be stored on my lab's R drive and on my own personal laptop. The R drive is the shared network that can be accessed directly through the lab's computers or if given permission, through a personal computer. That way my data is stored in two completely seperate sources. The R drive will contain files from Word, Excel, as well as other formats such as specific files that come from PCR macheines that have another version with the same informaitno on an excel and a word document. We have multiple copies in different formats of the information we collect, mainly for DNA and genotyping. My metadata and thesis will be saved on my perosnal laptop, as well as googledrive, but not on the R drive. In these ways, I have at least two copies of each source of information I need for my research and thesis. I am not sure if there is a way to edit lets say the google doc version of my thesis and have it automatically update the word version on y laptop, but as of now i have been copying and pasting the most recent version to the other, or i can download the google doc version under the same file name as my thesis on my laptop and just replace it. The shared network is backed up by Roots IT Support which contains a back up drive for all information if needed.

Any information on my laptop will be saved forever and will be backed up again using a "time machine". The informaiton on the R drive will be saved for at least a few years, but might be condensed in a few years and only the important documents are maintained but as a breeding program, files tend to be saved to look back on information on different varieties. 

# Access and data sharing
There are no factors that will restrict the sharing of the data my group has produced. Although we may not publish all of our data, if someone was curious, for example, how many herbicide resistant plants we had for each generation of breeding, we could easily look that up and share it. A chart of the other desired genes that were carried through each generation of breeding may also be created. The CoAxium Wheat was originally created by Colorado Wheat Research Foundation and owns the patent on the herbicide resistance gene. Therefore, my PI had to pay a trait fee to get seeds to work with but I do not have any restrictions for my project.

Some of the data will be publicly available when I publish my thesis. The rest will be available through a supplementary file or upon request. My thesis and research has to do with a potential new breeding scheme so documenting and its progression is the most important data that I can make that publicly available once I publish my thesis. Since we created our own KASP markers to run our DNA samples (and see which genes they carry) we can share the marker and primer data. In terms of policies governing the reuse, redistribution and creation of derivatives of the data/licenses, there will not be any policies for the data I generate. There is only a limitation on sharing the germplasm we produce with other breeding institutions. Anyone who requests to use the germplasm will need a license, similar to what we had to acquire at the start of our project.

There is a server (the “R” drive) that our lab has accessed to and is backed up and supported through ROOTS. I will have control and access of my data and will be available to others in my group as well. The data will be archived and preserved forever. The formats will be in Excel, Word and StepOne files (the format that the StepOne machine uses and detects the presence of certain genes/alleles, can also export data into Excel files). 




# Archiving and preservation

Repository with source code [here](https://github.com/clarallebot/GRAD521_DMPtemplate)
